utils/run_test.sh

```diff
@@ -99,21 +99,22 @@ if [ -d "/mnt/daos" ]; then
     run_test src/vos/tests/evt_ctl.sh pmem
     run_test "${SL_PREFIX}/bin/vea_ut"
     run_test src/rdb/raft_tests/raft_tests.py
-    # Satisfy CGO requirements for go-spdk bindings
-    export CGO_LDFLAGS="-L${SL_SPDK_PREFIX}/lib -L${SL_PREFIX}/lib64"
-    export CGO_CFLAGS="-I${SL_SPDK_PREFIX}/include"
+    go_spdk_ctests="build/src/control/lib/spdk/ctests/nvme_control_ctests"
+    echo "DEBUG: "
+    ls -lah "$go_spdk_ctests"
+    if test -f "$go_spdk_ctests"; then
+        run_test "$go_spdk_ctests"
+    else
+        echo "$go_spdk_ctests missing, SPDK_SRC not available when built?"
+    fi
```

Comment:
Wondering if this is enough extra logic to justify breaking it into another script, like run_go_tests.sh is.

Reply:
5lines so I would say no for the moment